### ISSUE TITLE
Improve Auto-Mappings to support more entities

### ISSUE THREAD

**Issue by maeg02** [bcolpaert/Colso.Xrm.DataTransporter#33]
Consider that you added a view to add entity name and entity key.
This could be:
entityname; Key:
TransactionCurrency; isocurrencycode
systemuser; internalemailaddress
account; accountnumber

and then users could add:
new_customentity; new_customattribute

this tool would then try to add auto mappings based on these entities and attributes

Could this be interesting?

**Comment by bcolpaert** [comment]
Yes, I was also thinking about a way to map based on entity attributes... I will try to include an attempt to do this in the next release!
